src/components/UriViewer/UriViewer.js

```diff
@@ -56,29 +65,95 @@ export const UriViewer = _.flow(
         setMetadata(metadata)
       }
     } catch (e) {
-      setLoadingError(await e.json())
+      // azure blob storage api only returns responses in xml format at this time
+      // https://feedback.azure.com/d365community/idea/0646f944-3a25-ec11-b6e6-000d3a4f0f84
+      if (isAzureUri(uri)) {
+        setLoadingError(await e.text())
+      } else {
+        setLoadingError(await e.json())
+      }
     }
   }
+
   useOnMount(() => {
     loadMetadata()
   })
 
-  const { size, timeCreated, updated, bucket, name, fileName, accessUrl } = metadata || {}
-  const gsUri = `gs://${bucket}/${name}`
-  const downloadCommand = getDownloadCommand(fileName, gsUri, accessUrl)
-  if (isAzureUri(uri)) {
+  const TerminalCommand = downloadCommand => h(Fragment, [
+    p({ style: { marginBottom: '0.5rem', fontWeight: 500 } }, [
+      'Terminal download command'
+    ]),
+    pre({
+      style: {
+        display: 'flex',
+        alignItems: 'center',
+        margin: 0,
+        padding: '0 0.5rem',
+        background: colors.light(0.4),
+      }
+    }, [
+      span({
+        style: {
+          overflowX: 'auto',
+          flex: '1 1 0',
+          padding: '1rem 0',
+        }
+      }, [downloadCommand || ' ']),
+      h(ClipboardButton, {
+        children: undefined,
```

Comment:
You don't need to define `children` and `onClick` here because they are undefined by default.

Please add an aria-label… I added a comment on an older commit apparently.